prisma/migrations/20230222171607_add_faucet_index/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- DropIndex
+DROP INDEX "index_users_on_graffiti";
```

Comment:
Can you remove the changes for graffiti on `users`? We have a trigram index on this, but Prisma is regenerating it

Reply:
done